debugger/debugrt/host/host.c

```diff
@@ -30,6 +31,85 @@ static void spin_unlock()
     InterlockedExchange(&_lock, 0);
 }
 
+static bool raise_debugger_events()
+{
+    static bool initialized = false;
+
+    if (IsDebuggerPresent())
+    {
+        if (!initialized)
+        {
+            // Set oe_debugger_contract_version from the environment.
+            // Normally the debugger would set the value depending on
+            // the version it supports. Setting it from the environment
```

Comment:
For my understanding, what does it mean to use a specific version of the contract? There's no example of how the debugrt negotiates whether it supports a given contract version specified by either the debugger or the environment setting. What does the "contract version" cover, for instance? Just the set of events fired but not the globals or data structures? 

Depending on how that's scoped, it seems like debugrt must only support the versions it understands (i.e. exactly `oe_debugger_contract_version == 1`). For example, if debugger uses contract version 2 because it expects a new global to be present for it to manipulate but it does not, should it fire the debug events? Or do you envision that this is piecemeal (i.e. firing these events will always continue to work for any version >= 1 and other features light up by version?

Reply:
Thanks for the discussion. I have simplified the use of the variable.
The variable denotes the version of the contract supported by the SDK.
The debugger can introspect this variable to determine if it can debug an application; if it cannot, it is expected to advise the user to use a newer version of the debugger.